src/api/objects/ObjectAPI.js

```diff
@@ -437,6 +472,23 @@ ObjectAPI.prototype._toMutable = function (object) {
     return mutableObject;
 };
 
+/**
+ * Updates a domain object based on its latest persisted state. Note that this will mutate the provided object.
+ * @param {module:openmct.DomainObject} domainObject an object to refresh from its persistence store
+ * @returns {Promise} the provided object, updated to reflect the latest persisted state of the object.
+ */
+ObjectAPI.prototype.refresh = async function (domainObject) {
+    const refreshedObject = await this.get(domainObject.identifier);
+
+    if (domainObject.isMutable) {
+        domainObject.$refresh(refreshedObject);
+    } else {
+        utils.refresh(domainObject, refreshedObject);
```

Comment:
The change to `utils` seems to be missing? It seems that the cherry pick only brought across changes to ObjectAPI and not other files?

Reply:
added missing changes